clients/java/src/test/java/io/camunda/zeebe/client/impl/command/CreateProcessInstanceCommandImplTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.impl.command;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.gateway.protocol.GatewayOuterClass.CreateProcessInstanceRequest;
+import io.camunda.zeebe.gateway.protocol.GatewayOuterClass.ProcessInstanceCreationStartInstruction;
+import java.util.List;
+import org.junit.Test;
+
+public class CreateProcessInstanceCommandImplTest {
```

Comment:
:thinking: I like this unit test in general. But we also have the [CreateProcessInstanceTest](https://github.com/camunda/zeebe/blob/main/clients/java/src/test/java/io/camunda/zeebe/client/process/CreateProcessInstanceTest.java) that contain test cases for the create process instance command. 

Do you want to add a test case to `CreateProcessInstanceTest`?